cypress/e2e/wip/Wallets/addcryptoWallet.cy.js

```diff
@@ -75,7 +75,9 @@ function checkWalletAccountSwitcher(walletname) {
 
 describe('QATEST-98773 - Add crypto wallet account', () => {
   beforeEach(() => {
-    cy.c_login({ user: 'walletloginEmail' })
+    cy.c_createCRAccount({ country_code: 'aq' })
+    cy.c_login()
```

Comment:
If this will login to wallet UI then do this but of it logs in to the tradershub UI then can leave it as it is
```suggestion
    cy.c_login({ app: 'wallets' })
```